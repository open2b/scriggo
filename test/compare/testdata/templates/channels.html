{# render #}

{% notifications := make(chan string, 10) %}
notifications: {{ not not notifications }}
len(notifications): {{ len(notifications) }}

{% if notifications %}
    BAD
{% else %}
    not elements to read
{% end if %}

{% notifications <- "message" %}
notifications: {{ not not notifications }}
len(notifications): {{ len(notifications) }}

{{ <-notifications }}

notifications: {{ not not notifications }}
len(notifications): {{ len(notifications) }}