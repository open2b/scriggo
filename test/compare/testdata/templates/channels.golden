notifications: false
len(notifications): 0

    not elements to read

notifications: true
len(notifications): 1

message

notifications: false
len(notifications): 0